playbooks/terraform-provider-openstack-acceptance-test-public-clouds/post.yaml

```diff
@@ -0,0 +1,61 @@
+- hosts: all
+  become: yes
+  roles:
+    - export-cloud-openrc
+  tasks:
+    - name: Clean up resources for public clouds & terraform-provider-openstack acceptance tests
+      shell:
+        cmd: |
+          set -e
+          set -x
+          # clean up instances
+          for ins_id in `openstack server list -f value -c ID -c Name |grep instance_1 | awk -F ' ' '{print $1}'`
+          do
+              openstack server delete "${ins_id}"
+          done
+
+          # clean up keypairs
+          for kp_name in `openstack keypair list -f value -c Name |grep kp_1 | awk -F ' ' '{print $1}'`
+          do
+              openstack keypair delete "${kp_name}"
+          done
+
+          # clean up security groups
+          for sg_id in `openstack security group list -f value -c ID -c Name |grep -E 'security_group|security_group_1|security_group_2' | awk -F ' ' '{print $1}'`
```

Comment:
`security_group|security_group_1|security_group_2` can be abbreviated as `security_group`